<ui version="4.0" >
 <class>SignatureDialog</class>
 <widget class="QWidget" name="SignatureDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>335</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="SignatureDialogLayout" >
   <item>
    <widget class="QLabel" name="title" />
   </item>
   <item>
    <widget class="QLabel" name="error" />
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="signer" >
      <attribute name="title" >
       <string>Signer</string>
      </attribute>
      <layout class="QGridLayout" name="signerLayout" >
       <item rowspan="3" row="0" column="0" >
        <widget class="QGroupBox" name="signerLocationGroup" >
         <property name="title" >
          <string>Signer location</string>
         </property>
         <layout class="QFormLayout" name="signerLocationGroupLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="signerCityLabel" >
            <property name="text" >
             <string>City</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="signerCity" >
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="signerStateLabel" >
            <property name="text" >
             <string>State</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="signerState" >
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="signerCountryLabel" >
            <property name="text" >
             <string>Country</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLineEdit" name="signerCountry" >
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="signerZipLabel" >
            <property name="text" >
             <string>Zip</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QLineEdit" name="signerZip" >
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1" >
           <spacer name="verticalSpacer" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>130</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QGroupBox" name="signerRoleGroup" >
         <property name="title" >
          <string>Role / resolution</string>
         </property>
         <layout class="QVBoxLayout" name="signerRoleGroupLayout" >
          <item>
           <widget class="QLineEdit" name="signerRole" >
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1" >
        <spacer name="signerSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>131</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" >
        <widget class="QPushButton" name="signerShowCert" >
         <property name="text" >
          <string>Show certificate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="signature" >
      <attribute name="title" >
       <string>Signature</string>
      </attribute>
      <layout class="QGridLayout" name="signatureLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QTreeWidget" name="signatureView" >
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <column>
          <property name="text" >
           <string>Attribute</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Value</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer name="signatureSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>264</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="signatureShowCert" >
         <property name="text" >
          <string>Show certificate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verify" >
      <attribute name="title" >
       <string>Verify</string>
      </attribute>
      <layout class="QGridLayout" name="verifyLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QTreeWidget" name="verifyView" >
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <column>
          <property name="text" >
           <string>Attribute</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Value</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer name="verifySpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>274</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="verifyShowCert" >
         <property name="text" >
          <string>Show certificate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>SignatureDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>199</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>signatureShowCert</sender>
   <signal>clicked()</signal>
   <receiver>SignatureDialog</receiver>
   <slot>showCertificate()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>335</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>signerShowCert</sender>
   <signal>clicked()</signal>
   <receiver>SignatureDialog</receiver>
   <slot>showCertificate()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>300</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verifyShowCert</sender>
   <signal>clicked()</signal>
   <receiver>SignatureDialog</receiver>
   <slot>showCertificate()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>335</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showCertificate()</slot>
 </slots>
</ui>
